broadlink/__init__.py

```diff
@@ -86,6 +86,7 @@
     0x2722: (S1C, "S2KIT", "Broadlink"),
     0x4ead: (hysen, "HY02B05H", "Hysen"),
     0x4e4d: (dooya, "DT360E-45/20", "Dooya"),
+    0x4f6e: (dooya_new, "DT360E-45/20", "Dooya"),
```

Comment:
Where did you get the model name from?

Reply:
The sticker on the motor housing and on packaging
![dooya_dt360](https://user-images.githubusercontent.com/55150081/94475814-5117a580-01d8-11eb-9ec4-0601a30baa3b.jpg)

Reply:
Beautiful.